src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -0,0 +1,259 @@
+  val cc0 = "http://creativecommons.org/publicdomain/zero/1.0"
+
+  def apply(emd: EasyMetadataImpl, audiences: Seq[String]): Try[Elem] = Try {
+    //    println(new EmdMarshaller(emd).getXmlString)
+
+    val dateMap: Map[String, Iterable[Elem]] = getDateMap(emd)
+    val dateCreated = dateMap("created")
+    val dateAvailable = {
+      val elems = dateMap("available")
+      if (elems.isEmpty) dateCreated
+      else elems
+    }
+   <ddm:DDM
+     xmlns:dc="http://purl.org/dc/elements/1.1/"
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xmlns:dct="http://purl.org/dc/terms/"
+     xmlns:dcx-dai="http://easy.dans.knaw.nl/schemas/dcx/dai/"
+     xmlns:dcx-gml="http://easy.dans.knaw.nl/schemas/dcx/gml/"
+     xmlns:gml="http://www.opengis.net/gml"
+     xmlns:abr="http://www.den.nl/standaard/166/Archeologisch-Basisregister/"
+     xmlns:ddm={schemaNameSpace}
+     xmlns:id-type="http://easy.dans.knaw.nl/schemas/vocab/identifier-type/"
+     xsi:schemaLocation={s"$schemaNameSpace $schemaLocation"}
+   >
+     <ddm:profile>
+       { emd.getEmdTitle.getDcTitle.asScala.map(bs => <dc:title xml:lang={ lang(bs) }>{ bs.getValue }</dc:title>) }
+       { emd.getEmdDescription.getDcDescription.asScala.map(bs => <dct:description xml:lang={ lang(bs) }>{ bs.getValue }</dct:description>) }
+       { /* instructions for reuse not specified as such in EMD */ }
+       { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator>{ bs.getValue }</dc:creator>) }
+       { emd.getEmdCreator.getEasCreator.asScala.map(author => <dcx-dai:creatorDetails>{ toXml(author)} </dcx-dai:creatorDetails>) }
+       { dateCreated.map(node =>  <ddm:created>{ node.text }</ddm:created>) }
+       { dateAvailable.map(node =>  <ddm:available>{ node.text }</ddm:available>) }
+       { audiences.map(code => <ddm:audience>{ code }</ddm:audience>) }
+       <ddm:accessRights>{ emd.getEmdRights.getAccessCategory }</ddm:accessRights>
+     </ddm:profile>
+     <ddm:dcmiMetadata>
+       { emd.getEmdIdentifier.getDcIdentifier.asScala.filter(isDdmId).map(bi => <dct:identifier xsi:type={ idType(bi) }>{ bi.getValue }</dct:identifier>) }
+       { emd.getEmdTitle.getTermsAlternative.asScala.map(str => <dct:alternative>{ str }</dct:alternative>) }
+       { emd.getEmdDescription.getTermsAbstract.asScala.map(bs => <ddm:description xml:lang={ lang(bs) } descriptionType='Abstract'>{ bs.getValue }</ddm:description>) }
+       { emd.getEmdDescription.getTermsTableOfContents.asScala.map(bs => <ddm:description xml:lang={ lang(bs) } descriptionType='TableOfContent'>{ bs.getValue }</ddm:description>) }
+       { emd.getEmdRelation.getDCRelationMap.asScala.map { case (key, values) => values.asScala.filterNot(_.getScheme == "STREAMING_SURROGATE_RELATION").map(toRelationXml(key, _)) } }
+       { emd.getEmdRelation.getRelationMap.asScala.map { case (key, values) => values.asScala.map(toRelationXml(key, _)) } }
```

Comment:
Are you skipping the `STREAMING_SURROGATE_RELATION` elements entirely, or do they come with the `getRelationMap`? Please don't forget about those!